### PR TITLE
Tweak md breakpoint

### PR DESCRIPTION
**Pull request by buberdds**
Don's ideas from Slack, ~waiting for deploy URL~

### CODE REVIEW

File: .changelog/532.bugfix.2.md [hunk 1]
```diff
@@ -0,0 +1 @@
+Increase md breakpoint value
```
Comment (csillag) [inline .changelog/532.bugfix.2.md:1]: Why do we have two changelog fragments, instead one, with two lines? (Is this how it's supposed to work?)

File: src/styles/theme/defaultTheme.ts [hunk 2]
```diff
@@ -82,6 +82,15 @@ declare module '@mui/material/styles' {
 }
 
 export const defaultTheme = createTheme({
+  breakpoints: {
+    values: {
+      xs: 0,
+      sm: 600,
+      md: 1000, // default is 900
```
Comment (csillag) [inline src/styles/theme/defaultTheme.ts:89]: Do we have to override all of them, in order to override this single one? Can't we do something like 
```
values: {
   ...defaultValues,
   md: 900,
}
```

?

File: .changelog/532.bugfix.2.md (hunk 1 above)
Comment (buberdds) [reply .changelog/532.bugfix.2.md:1]: I am following one of the comment I've received in the past 
https://github.com/oasisprotocol/oasis-core-ledger/pull/261#discussion_r1174122546

**Review by tjanez — APPROVED** [review]

**Comment by csillag** [comment]
Is the W.I.P in the title still relevant?

**Pull request merged**